vm-image-spec.yaml

```diff
@@ -287,6 +287,16 @@ files:
           - checkpoints_timed
         query: |
           SELECT checkpoints_req, checkpoints_timed FROM pg_stat_bgwriter;
+
+      # Number of slots is limited by max_replication_slots, so collecting position for all of them shouldn't be bad.
+      - metric_name: logical_slot_restart_lsn
+        type: gauge
+        help: 'restart_lsn of logical slots'
+        key_labels:
+          - slot_name
+        values: [restart_lsn]
+        query: |
+          select slot_name, restart_lsn from pg_replication_slots where slot_type = 'logical';
```

Comment:
How does it handle queries with multiple results?

Reply:
https://github.com/free/sql_exporter/blob/6f96b0d6b40340929e4d84c902378e7ead113ff1/documentation/sql_exporter.yml#L52

but I also learned this several hours ago, not an expert

Reply:
> How does it handle queries with multiple results?

@save-buffer See `pg_stats_userdb` query example